pvlib/modelchain.py

```diff
@@ -379,10 +379,19 @@ def dc_model(self, model):
                                          str(missing_params))
                 if model == 'sapm':
                     self._dc_model = self.sapm
-                elif model == 'singlediode':
-                    self._dc_model = self.singlediode
+                elif model == 'desoto':
+                    self._dc_model = self.desoto
+                elif model == 'pvsyst':
+                    self._dc_model = self.pvsyst
                 elif model == 'pvwatts':
                     self._dc_model = self.pvwatts_dc
+                elif model == 'singlediode':
+                    warnings.warn('DC model keyword singlediode used for '
+                                  'ModelChain object. singlediode is '
+                                  'ambiguous, use desoto instead. singlediode '
```

Comment:
desoto or pvsyst?

Reply:
Desoto for backwards compatibility, that's what `singlediode` has meant to date.